cleverpush/src/main/java/com/cleverpush/stories/models/Content.java

```diff
@@ -0,0 +1,67 @@
+
+package com.cleverpush.stories.models;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+import java.io.Serializable;
+
+public class Content implements Serializable {
+
+    @SerializedName("version")
+    @Expose
+    private Integer version;
+    @SerializedName("supportsLandscape")
+    @Expose
+    private Boolean supportsLandscape;
+    @SerializedName("title")
+    @Expose
+    private String title;
+    @SerializedName("canonicalUrl")
+    @Expose
+    private String canonicalUrl;
+    @SerializedName("preview")
+    @Expose
+    private Preview preview;
+
+    public Integer getVersion() {
+        return version;
+    }
+
+    public void setVersion(Integer version) {
+        this.version = version;
+    }
+
+    public Boolean getSupportsLandscape() {
```

Comment:
why Boolean object instead of boolean (datatype)?

Reply:
also "getters" for booleans are not supposed to be "getted"
java conventions state its either "is" or "has"

so
"isSupportsLandscape()" would be correct